### PR TITLE
Made use of Travis token caching within conda-smithy.

### PR DESCRIPTION
**Pull request by pelson**

### CODE REVIEW

**Comment by jakirkham** [comment]
Guess that works. Also unclear on when it goes stale. Was able to use mine for ~16hrs before I deleted it. The environment variable seems ok.

Should we really cache it though? It seems like that is a potential security risk.

**Comment by jakirkham** [comment]
Ah, sorry, for a second I thought we were caching the raw token on Travis CI. Clearly I have misread. This seems reasonable. I remain concerned about how quickly it goes stale. However, I'm ok with finding out this way.

**Comment by jakirkham** [comment]
I'm running into Travis CI API issues again. Figure let's go ahead and try this. If doesn't work, it is easy enough to revert. However, if it works, it will save me a headache.

File: .CI/create_feedstocks [hunk 1]
```diff
@@ -10,5 +10,8 @@ if [ -n "$GH_TOKEN" ]; then
     conda config --add channels conda-forge
     conda install --yes --quiet conda-smithy
 
+    mkdir -P ~/.conda-smithy
```
Comment (jakirkham) [inline .CI/create_feedstocks]: Oops, this should have been `-p`. Will fix.

Comment (jakirkham) [inline .CI/create_feedstocks]: See this PR ( https://github.com/conda-forge/staged-recipes/pull/1124 ).

**Comment by jakirkham** [comment]
Temporarily disabling this with PR ( https://github.com/conda-forge/staged-recipes/pull/1125 ) as the token appears to be expired. Probably because we got a new one when restarting Travis CI. Should try to coordinate this effort tomorrow.

**Comment by pelson** [comment]
Reverted in https://github.com/conda-forge/staged-recipes/pull/1127

File: .travis.yml [hunk 2]
```diff
@@ -17,6 +17,9 @@ env:
...
 
+        # Add the TRAVIS_TOKEN environment variable for the "conda-forge" Travis user.
+        - secure: "BBJrw5OcEG4zEUxjXt2txNoz+o7ilTb0e8Tj1626USKqi6HAWOdSFHW7PLYR6vyefWsHnGLAeYMRr9UB56O2oDwS7jgClBKbyut2+Q+8SQBwEtNCMHdU996IzSuDP4AM3h7DLjTsuinWhwpqLu72z2ZjE9OMz4q03SyyiBOecWU1btYbp/u3bpNzyhb2u0HMkRUnS8MxPUa8aHZBKp/JpLrGQmIT0LcjLNp4/jTZp+cs+J//02hAhJHV3vMmyefVvRZayu3xKzutpb3qIga0JAP3d0j2FWXRHVTW6Ke5H4u/4NU7iTpdRjpymnzH7H2kfOrNzgCJ23NfNhScDJDc5URvdXlstgkC4lDkd7BbSVgKKaVy2vVcMkz98qbBnhi0akMHNZXPLvT9tec9AsAX5/pdNaa1rH/iRhjWRfTCnqODX+RTwMhA1/3rkmhzI/8JFCcaZet8+lA2LtOf95jGmwcRwPlNFX0Jdd1QBk2nUAUpCePWIdg7GRxYuO7a4QCbhc6bSyB/WrCQZ1oEmtwGSRqxpMKvQmjxJObg40MgP0DaquGov30A4QLoEdvXCnM7w3QyIr5L/BrLlnCy8LesAZIBn2l5LAHrBFGh9t/lCK9APlUjD9dgT6eCYOSbfdBghKCGsajAsM/fwsPvcjOmpFCLf+UZTer36B57ccHrYdU="
```
Comment (jakirkham) [inline .travis.yml]: FYI. Seems we are not the only people doing [this]( https://github.com/grosser/travis_dedup/blame/041373bbe8e07cfa76b5ce96bc21e0d9fe077471/Readme.md#L37 ).

**Comment by jakirkham** [comment]
I think we need to generate a new Travis token. staged-recipes is getting really blocked up and rarely ever converting. Any chance you would have time to help, @pelson? I don't think I have access to the right user.

**Pull request merged**